File: Python/ex13-soma dos fatoriais.py
def fatorial(k):
    acumulador = 1

    for i in range(1, k + 1):
        acumulador *= i

    return acumulador

def soma(x):
    acumulador2 = 0

    for k in range(1, x + 1):
        acumulador2 += fatorial(k)

    return (acumulador2 + 1)

n = int(input())

print(soma(n))

File: exercicios_e_aulas/ex13-soma_dos_fatoriais.c
#include<stdio.h>

int fatorial(int k){
    int acumulador = 1;

    for(int i = 1; i <= k; i++){
        acumulador *= i;
    }
    
    return acumulador;
}

int soma(int x){
    int acumulador2 = 0;

    for(int i = 1; i <= x; i++){
        acumulador2 += fatorial(i);
    }

    return (acumulador2 + 1);
}

int main(int argc, char *argv[]){
    int n;
    scanf("%d", &n);

    int resposta = soma(n);

    printf("%d", resposta);
}